lib/datadog/lambda.rb

```diff
@@ -176,10 +186,18 @@ def self.initialize_listener
         Datadog::Utils.logger.debug("Setting merge traces #{merge_xray_traces}")
       end
 
-      Trace::Listener.new(handler_name: handler,
-                          function_name: function,
-                          patch_http: @patch_http,
-                          merge_xray_traces: merge_xray_traces)
+      # Only initialize listener if Tracing enabled.
+      unless Datadog::Tracing.enabled?
```

Comment:
is enabled? available on all classes in ruby or specifically on Datadog::Tracing?